.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
 repos:
+    -   repo: https://github.com/python-poetry/poetry
+        rev: 1.6.1
+        hooks:
+            -   id: poetry-lock
```

Comment:
A lock here is good as the lock file would have to be committed if it gets updated before pre-commit would pass it.